### ISSUE TITLE
Prebuilt binaries for NodeJS 22

### ISSUE THREAD

**Issue by jkowalleck** [marudor/libxmljs2#209]
Please add prebuilt binaries for NodeJS 22.

**Comment by bantao-kainos** [comment]
Yes please much urgently needed prebuilt binaries for latest node LTS version -- node@22.1.0

**Comment by bantao-kainos** [comment]
Also there are 2 high vulnerabilities detected in the last built CVE-2024-34393 and CVE-2024-34394
